### ISSUE TITLE
Result replication for cifar100

### ISSUE THREAD

**Issue by rahulvigneswaran** [naver-ai/cmo#1]
I used the code provided, fixed the seed(=1) and ran three experiments,
- CE
- CE + CMO
- BS + CMO

|Experiment|Accuracy from your code|Accuracy from paper (Table 2)|
|--------|---------|---------|
|CE|38.51|38.6|
|CE + CMO|35.03|43.9|
|BS + CMO|29.76|46.6|

I used the following code to run the experiments,
```sh
python cifar_train.py --dataset cifar100 --loss_type CE --epochs 200 --seed 1 --data_aug vanilla --gpu 1 

python cifar_train.py --dataset cifar100 --loss_type CE --epochs 200 --seed 1 --data_aug CMO --gpu 2 

python cifar_train.py --dataset cifar100 --loss_type BS --epochs 200 --seed 1 --data_aug CMO --gpu 3 

```

> Other than fixing the seed, I didn't make any changes to the code from the repo.

**Comment by pseulki** [comment]
Hello, Thank you for your interest! 

After I read your issue, I newly cloned git, and ran your codes (and additionally with seed=2) .
However, I got results as follows:

 | Method | Seed =1 | Seed=2| 
 | :---:  | :---: | :---: |  
 | CE + CMO  | 43.13 | 42.63 | 
 | BS + CMO  | 46.18 | 46.34 |

So I'm not sure why there is so much performance difference when you ran the experiment.
I can send you my training/test logs! :)

Here are my experimental setting and I hope you can find some clues.

My server status: 
- Intel(R) Core(TM) i9-9900X CPU @ 3.50GHz
- 4 GeForce GTX 1080 Ti GPUS 
- pytorch: 1.6, torchvision: 0.7, numpy: 1.18

If you have further questions, please feel free to discuss with us.
I hope to find the problem and solve your case.

**Comment by rahulvigneswaran** [comment]
Sorry, I checked my logs. For some reason, the experiments terminated well before 200 epochs. Seems to be a localized issue. I re-ran the experiments again and got the following.

|Experiment|Accuracy from your code (Seed = 1)|Accuracy from paper (Table 2)|
|--------|---------|---------|
|CE|38.81|38.6|
|CE + CMO|42.15|43.9|
|BS + CMO|45.48|46.6|

Thanks Alot!
